RELEASE.md

```diff
@@ -8,32 +8,42 @@ This release process targets the following goals:
    the corresponding artifacts.
 1. Ensure there exists a clear _separation of duties_ between those who _write_
    the code and those who _release_ it (and announce its release).
+1. Ensure that, under no circumstances, a single individual or entity is a
+   single point of (security) failure.
```

Comment:
Maybe something along the lines of:

```suggestion
1. Ensure that, under no circumstances, a single individual or entity is a
   single point of (security) failure or otherwise unanimously compromise the security of the sBTC protocol or its projects.
```